docs/tiling.md

```diff
@@ -0,0 +1,337 @@
+- **`minzoom` and `maxzoom`:** Defines the range of zoom levels for which the tiles are available. 
+- **`bounds`:** Optional. Specifies the geomoetrical bounds covered by the tileset. Specified as an array of four numbers in the order `[minX, minY, maxX, maxY]`.
+- **`center`:** Optional. Indicates the center and suggested default view of the tileset.
+- **`format`:** Required. The format of the tiles, for the purpose of this specification, we are currently using `json`, but are planning to extend this to binary vector format in the future.
+- **`vector_layers`:**  Required. Describes each layer within the vector tiles, and has the following structure:
+
+  - **`id`:** Required. A unique identifier for the layer. Required for each layer.
+  - **`fields`:** Required. A list of fields (attributes) and their data types. For MicroJSON, this can either be an empty list, or any combination of `string`, `numerical`, and `multi-numeric`, attributes with their sub-attributes, as described in the MicroJSON specification. Required for each layer.
+  - **`description`:** Optional. A brief description of the layer.
+  - **`minzoom` and `maxzoom`:** Optional. The range of zoom levels at which the layer is visible.
+- **`fillzoom`:** Optional. An integer specifying the zoom level from which to generate overzoomed tiles.
+- **`legend`:** Optional. Contains a legend to be displayed with the tileset.
+
+The following fields of TileJSON may be used if the use case requires it, and are included here for completeness:
+
+- **`scheme`:** The tiling scheme of the tileset.
+- **`grids`:** The URL pattern for accessing grid data.
+- **`data`:** Optional. The URL pattern for accessing data. Used for GeoJSON originally, which in this specification is replaced by MicroJSON and used in the `tiles` field.
+- **`template`:** Optional. Contains a mustache template to be used to format data from grids for interaction.
+
+## Pydantic Model for TileJSON for MicroJSON
+### TileJSON
+::: microjson.tile.TileJSON
+### TileModel
+::: microjson.tile.TileModel
+### TileLayer
+::: microjson.tile.TileLayer
+
+## TileJSON for MicroJSON example with Vector Layers
+The below example shows a TileJSON object for a MicroJSON tileset with multiple layers of detail. The tileset has a single vector layer, `image_layer` id of `vector_layers`, which contains a single vector layer describing images. The `fields` property of the this layer specifies the attributes of the layer, including the data types of the attributes. The `tiles` property specifies the URL pattern for accessing the vector tiles, which in this case is a 2D data set (no channels, time or z-axis) with zoom level. The `fillzoom` property specifies the zoom level from which to generate overzoomed tiles, which in this case starts at level 3, after the last specified layer.
```

Comment:
The below example illustrates a TileJSON for a MicroJSON tiles containing multiple layers of detail

Reply:
Changed, but I think it makes sense to keep the term "tileset" for a set of tiles.